AK/NumberFormat.h

```diff
@@ -42,7 +42,29 @@ static inline String human_readable_size_long(u64 size)
         return String::formatted("{} ({} bytes)", human_readable_size(size), size);
 }
 
+static inline String human_readable_time(long long time_in_seconds)
```

Comment:
We generally stray away from types like `long long`, please use `i64` instead (as that's what `AK::Time::to_seconds` returns), or templatize this function.